backend/config/__init__.py

```diff
@@ -151,6 +151,13 @@ def str_to_bool(value: str) -> bool:
     "SCHEDULED_UPDATE_LAUNCHBOX_METADATA_CRON",
     "0 5 * * *",  # At 5:00 AM every day
 )
+ENABLE_SCHEDULED_CONVERT_IMAGES_TO_WEBP: Final = str_to_bool(
+    os.environ.get("ENABLE_SCHEDULED_CONVERT_IMAGES_TO_WEBP", "false")
+)
+SCHEDULED_CONVERT_IMAGES_TO_WEBP_CRON: Final = os.environ.get(
+    "SCHEDULED_CONVERT_IMAGES_TO_WEBP_CRON",
+    "0 6 * * *",  # At 6:00 AM every day
```

Comment:
nitpick: Probably something for later, but we could put all tasks unrelated to Scan into the same hour.
I feel otherwise, as we add more tasks, it will make the system to have scheduled processes running for most part of the day (if using default values).